extension/source/QuillPage/Wallet/Wallets/AssetsTable.tsx

```diff
@@ -0,0 +1,160 @@
+// @ts-nocheck // TODO: fix types
+import * as React from 'react';
+import { useTable, usePagination } from 'react-table';
+import {
+  CurrencyEth,
+  ShareNetwork,
+  StopCircle,
+  CurrencyDollar,
+  CaretLeft,
+  CaretRight,
+} from 'phosphor-react';
+import { TokenData } from './WalletDetail';
+
+export const TableHeader: React.FunctionComponent = () => {
+  return (
+    <div className="flex justify-between p-6">
+      <div className="flex gap-4">
+        <div className="flex gap-2 items-center">
+          <ShareNetwork className="text-blue-500 icon-md" />
+          <div className="">
+            <span className="text-blue-500">Network :</span> Mainnet
+          </div>
+        </div>
+        <div className="flex gap-2 items-center">
+          <StopCircle className="text-blue-500 icon-md" />
+          <div className=""> 3 Tokens</div>
+        </div>
+      </div>
+      <div className="flex gap-4 text-[13pt]">
+        <div className="flex gap-2 items-center">
+          <CurrencyEth className="text-blue-500 icon-md" />
+          <div className="">0.937 ETH</div>
+        </div>
+        <div className="flex gap-2 items-center">
+          <CurrencyDollar className="text-blue-500 icon-md" />
+          <div className="">3,813.38 USD</div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+interface IAssetsTable {
+  data: TokenData[];
+}
+
+export const AssetsTable: React.FunctionComponent<IAssetsTable> = ({
```

Comment:
Need to make this generic so single table can support Assets and Transactions 
tracking in #146